lstchain/io/lstcontainers.py

```diff
@@ -121,13 +121,10 @@ def set_disp(self, source_pos, hillas):
         self.disp_miss = disp.miss
 
     def set_timing_features(self, geom, image, peakpos, hillas):
-        try:
-            peak_time = Quantity(peakpos) * u.Unit("ns")
-            timepars = time.timing_parameters(geom, image, peak_time, hillas)
-            self.time_gradient = timepars['slope'].value
-            self.intercept = timepars['intercept']
-        except:
-            pass
+        peak_time = Quantity(peakpos) * u.Unit("ns")
```

Comment:
I agree that `except` should be handled differently than `pass`.
However, I might want to continue and get the dl1 container even without the reconstructed timing.

Reply:
I think it would be good to have an example event at hand, where the timing cannot be reconstructed. This way, the feature you want can be implemented and it can be ensured that this feature works forever. For me (and possibly for everybody) having such an example event would make it easier to understand what is failing exactly, so we can catch the correct exception. 

At the moment, I can only assume the failing part would be somewhere in
[`ctapipe.image.timing_parameters.timing_parameters()`](https://github.com/cta-observatory/ctapipe/blob/master/ctapipe/image/timing_parameters.py#L15), but looking at that code, I think this cannot fail unless the hillas parameters were not calculated, but we already catch that case. 

Anyway I might be guessing in the entirely wrong direction ... I assume you have an example event somewhere, for which the timing reconstruction fails?

Reply:
Thank you for replying.
I don't actually have an example in mind.
I think I am dealing with bare expects in a too broad sense.